src/main/java/ava/Ava.java

```diff
@@ -0,0 +1,232 @@
+            } else if (task.startsWith("todo")) {
+                try {
+                    addTask(tasks, task, "todo");
+                } catch (EmptyDescriptionException e) {
+                    dealWithEmptyDescriptionException("todo");
+                    continue;
+                }
+            } else if (task.startsWith("deadline")) {
+                try {
+                    try {
+                        addTask(tasks, task, "deadline");
+                    } catch (ArrayIndexOutOfBoundsException e) {
+                        dealWithFormatException();
+                        continue;
+                    }
+                } catch (EmptyDescriptionException e) {
+                    dealWithEmptyDescriptionException("deadline");
+                    continue;
+                }
+            } else if (task.startsWith("event")) {
+                try {
+                    try {
+                        addTask(tasks, task, "event");
+                    } catch (ArrayIndexOutOfBoundsException e) {
+                        dealWithFormatException();
+                        continue;
+                    }
+                } catch (EmptyDescriptionException e) {
+                    dealWithEmptyDescriptionException("event");
+                    continue;
+                }
+            } else if (task.startsWith("delete")) {
+                deleteTask(tasks, task);
+                DataFile.saveTasks(tasks);
+                continue;
+            } else {
+                dealWithUnknownException();
+                continue;
+            }
+            printAfterAddingTask(tasks);
+            DataFile.saveTasks(tasks);
+        }
+    }
+
+    private static void dealWithUnknownException() {
+        printLine();
+        System.out.println("(⊙_⊙)? I'm sorry!!! But I don't know what that means.");
+        printLine();
+    }
+
+    private static void dealWithFormatException() {
+        printLine();
+        System.out.println("(⊙_⊙)? You need to specify the date after '/'");
+        printLine();
+    }
+
+    private static void dealWithEmptyDescriptionException(String type) {
+        printLine();
+        switch (type) {
+        case "todo":
+            System.out.println("Please tell me what needs todo (＾＿－)");
+            break;
+        case "deadline":
+            System.out.println("Please tell me the deadline is for? (＾＿－)");
+            break;
+        case "event":
+            System.out.println("Please tell me what is the event? (＾＿－)");
+            break;
+        }
+        printLine();
+    }
+
+    public static void addTask(ArrayList<Task> tasks, String task, String type) throws EmptyDescriptionException {
+        task = task.replace(type, "");
+        if (task.isEmpty()) {
+            throw new EmptyDescriptionException();
+        }
+        String[] taskAndDate = task.split("/");
+        taskAndDate[0] = taskAndDate[0].trim();
+        switch (type) {
+        case "todo":
```

Comment:
Try separating logically linked statements by line breaks. This too enhances the readability of code.

```suggestion
        task = task.replace(type, "");
        
        if (task.isEmpty()) {
            throw new EmptyDescriptionException();
        }
        
        String[] taskAndDate = task.split("/");
        taskAndDate[0] = taskAndDate[0].trim();
        
        switch (type) {
        case "todo":
```